README_webgpu.md

```diff
@@ -47,7 +47,7 @@ JIT compilation is not supported when using WebGPU with WASM.
 
 ## Running natively: `HL_TARGET=host-webgpu`
 
-> _Tested with top-of-tree Dawn as of 2023-03-14._
+> _Tested with top-of-tree Dawn as of 2023-11-03._
```

Comment:
Can you add a short line in this section (perhaps before line 55?) that is essentially:
```
Note that as of 2023-10-31,
wgpu is not supported due to [lacking `override` support for wgsl](https://github.com/gfx-rs/wgpu/issues/1762)
which we require in order to set GPU block sizes.```

Reply:
Done.